setup.py

```diff
@@ -0,0 +1,26 @@
+from setuptools import setup
+from setuptools import find_packages
+
+import sdl2promela
+
+setup(
+    name="sdl2promela",
+    description="SDL 2 Promela translator, created within the scope of TASTE Model Checker Project financed by the European Space Agency",
+    version=sdl2promela.__version__,
+    packages=find_packages(),
+    author="Michal Kurowski",
```

Comment:
UTF-8 is supported ;)
(ł)

Reply:
It you say so... but it will fail on some systems...